packages/web3-core-helpers/src/formatters.js

```diff
@@ -219,6 +219,7 @@ var outputTransactionReceiptFormatter = function (receipt){
     if(typeof receipt.status !== 'undefined' && receipt.status !== null) {
         receipt.status = Boolean(parseInt(receipt.status));
     }
+    console.log('using new web3 at to format transaction receipt')
```

Comment:
Edit this file to remove this line. It proves that the bundle is being published/used correctly [in travis here](https://travis-ci.com/cgewecke/web3.js/jobs/251293678#L1669-L1676).